base/NCIO.F90

```diff
@@ -5569,6 +5571,11 @@ subroutine MAPL_ReadTilingASCII(layout, FileName, GridName, NT, im, jm, n_Grids,
           allocate(AVR(NT,9), STAT=STATUS) ! 9 columns for EASE grid
           _VERIFY(STATUS)
           allocate(AVR_transpose(9,NT))
+          ! WY note: In the old tile file, 
+          ! EASE grid name is in the form of SMAP-EASEvx-Mxx
+          ! It is changed to a standard form as EASEvx_Mxx
+          Correct_ease_name = ease_grid_name(IM(1))
+          GridNAME(1) = Correct_ease_name
```

Comment:
I'm concerned about going through the number of cols to identify the grid.  See my comment near Line ~788 of MAPL_EASEConversion.F90.  Here, if we encounter the old naming convention ("SMAP-EASE..."),  it would be simple to directly convert the string SMAP-EASEv[x]-M[xx] into EASEv[x]_M[xx].